extensions/adobe/experience/decisioning/placement-detail.schema.json

```diff
@@ -0,0 +1,51 @@
+{
+  "meta:license": [
+    "Copyright 2019 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id": "https://ns.adobe.com/experience/decisioning/placement-detail",
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "title": "Decision Placement Detail",
+  "type": "object",
+  "meta:extensible": true,
+  "meta:abstract": true,
+  "description": "Detail about the entity defining and controlling the decisioning process. Decisions are made in the context of a placement. At any given time, the best option is re-evaluated, or ranked, based on the most current set of context variables, rules and constraints applicable for this placement.",
+  "definitions": {
+    "placement-detail-datatype": {
+      "properties": {
+        "xdm:id": {
+          "title": "Placement Identifier",
+          "type": "string",
+          "format": "uri-reference",
+          "description": "A unique identifier for the decision placement."
+        },
+        "xdm:version": {
+          "title": "Placement Version",
+          "type": "string",
+          "description": "The version that the placement object was at when the snapshot was taken."
+        },
+        "xdm:name": {
+          "title": "Placement Name",
+          "type": "string",
+          "description": "Placement name. The name is displayed in various user interfaces."
+        },
+        "xdm:channel": {
```

Comment:
Different definition for xdm:channel - https://github.com/adobe/xdm/blob/master/schemas/context/experienceevent-channel.schema.json#L19

Reply:
The difference is that the experience event mixin inlines the definition and the placement references the channel. Since channel has an `@id` property it should not really be "inlined", otherwise you end up with a JSON object that has multiple `@id`s. According a discussion with Leonard we should not embed/inline an entity schema in another entity schema.
If you ask me it should be a string whose value is that of the channel entity's `@id` property.